### DISCUSSION TITLE
Adjusting the style of the Branding Title

### DISCUSSION THREAD

**Discussion by randyDarsh** [mui/toolpad#4469] [Help and support]
It's not clear how to change the styling of the branding title prop.  Inspecting the element, it appears to be a H6 typography tag, but I can't seem to make any meaningful changes to its specific styling. Anyone explain how to adjust the styling of the title prop?

**Comment by bharatkashyap** [comment]
For the next release, we're working on adding a `slot` and `slotProps` to be able to customise this section deeply as part of https://github.com/mui/toolpad/issues/4342

**Reply by randyDarsh** [reply]
Sounds good.  Looking forward to it.
